sos/plugins/sssd.py

```diff
@@ -29,6 +29,7 @@ def setup(self):
         self.add_copy_spec([
             "/etc/sssd/sssd.conf",
             "/var/log/sssd/*",
+            "/var/lib/sss/pubconf/krb5.include.d/*",
```

Comment:
No passwords or secret information stored here, if you would like I can ask a SSSD developer to confirm this.